app/lib/fullUrlUtils.js

```diff
@@ -0,0 +1,15 @@
+function url(req) {
+  const protocol = req.protocol;
+  const host = req.get('host');
```

Comment:
Is there a reason this doesn't use `req.hostname`?
http://expressjs.com/en/4x/api.html#req.hostname

Reply:
I read somewhere that you should use `req.get('host');`. However, I've tried your change and it works a treat.